tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,149 @@
+  SECTION(section_name) {
+    REQUIRE(
+        std::is_same_v<sycl::accessor<T>, sycl::accessor<T, expected_dims>>);
+  }
+  section_name = "Verify default value for accessMode template parameter for " +
+                 type_name + " data type. [generic accessor]";
+  SECTION(section_name) {
+    REQUIRE(
+        std::is_same_v<sycl::accessor<T, expected_dims>,
+                       sycl::accessor<T, expected_dims,
+                                      std::is_const_v<T>
+                                          ? sycl::access_mode::read
+                                          : sycl::access_mode::read_write>>);
+  }
+  section_name = "Verify default value for accessMode template parameter for " +
+                 type_name + " data type. [generic accessor]";
+  SECTION(section_name) {
+    REQUIRE(
+        std::is_same_v<sycl::accessor<T, expected_dims>,
+                       sycl::accessor<T, expected_dims,
+                                      std::is_const_v<T>
+                                          ? sycl::access_mode::read
+                                          : sycl::access_mode::read_write>>);
+  }
+  section_name =
+      "Verify default value for accessTarget template parameter for " +
+      type_name + " data type. [generic accessor]";
+  SECTION(section_name) {
+    REQUIRE(std::is_same_v<
+            sycl::accessor<T, expected_dims,
+                           std::is_const_v<T> ? sycl::access_mode::read
+                                              : sycl::access_mode::read_write>,
+            sycl::accessor<T, expected_dims,
+                           std::is_const_v<T> ? sycl::access_mode::read
+                                              : sycl::access_mode::read_write,
+                           expected_target>>);
+  }
+  section_name =
+      "Verify default value for isPlaceholder template parameter for " +
+      type_name + " data type. [generic accessor]";
+  SECTION(section_name) {
+    REQUIRE(std::is_same_v<
+            sycl::accessor<T, expected_dims,
+                           std::is_const_v<T> ? sycl::access_mode::read
+                                              : sycl::access_mode::read_write,
+                           expected_target>,
+            sycl::accessor<T, expected_dims,
+                           std::is_const_v<T> ? sycl::access_mode::read
+                                              : sycl::access_mode::read_write,
+                           expected_target, expected_placeholder>>);
+  }
+}
+
+/**
+ * @brief Struct with functor that will be used in type coverage to call all
+ *        verification functions.
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+template <typename T>
+class run_tests {
+ public:
+  void operator()(const std::string& type_name) {
+    test_for_generic_acc<T>(type_name);
+    test_for_host_acc<T>(type_name);
+    test_exception_for_local_acc<T>(type_name);
```

Comment:
```suggestion
    const auto const_type_name = "const " + type_name;
    test_for_generic_acc<T>(type_name);
    test_for_generic_acc<const T>(const_type_name);
    test_for_host_acc<T>(type_name);
    test_for_host_acc<const T>(const_type_name);
    test_exception_for_local_acc<T>(type_name);
    test_exception_for_local_acc<const T>(const_type_name);
```
Did I get the intent right?

Reply:
yes, coverage for const types has been added [here](https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/c040cc7560db878fb467d794a504c6e5e20a0d0e#diff-9ed3eae8f2889bc04dfc0fad247a8fa4e381e1fff4144e0f1a23670d108f6fe9R157-R162)